### ISSUE TITLE
AttributeError: module 'tensorflow' has no attribute '__version__'

### ISSUE THREAD

**Issue by elvys-zhang** [tensorflow/tensorflow#31576]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): centos 7.2
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): pip install tensorflow
- TensorFlow version (use command below): 1.10
- Python version: 3.6.7
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with: 1. TF 1.0: `python -c "import
tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` 2. TF 2.0: `python -c
"import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**
I installed tensorflow 1.10, and it worked fine. After I installed tensorflow-serving-api 1.10, error occured.   
```
Original exception was:
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'tensorflow' has no attribute '__version__'

```
And the result of dir(tf) is :
```
>>> dir(tf)
['__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__']
```
It seems that the "__init__.py" under tensorflow has been cleaned.   
Before installation of tf-serving-api: 
```
[root@eac5952e1443 /]# cat /usr/local/anaconda3/lib/python3.6/site-packages/tensorflow/__init__.py 
# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");

...
_names_with_underscore = ['__version__', '__git_version__', '__compiler_version__', '__cxx11_abi_flag__', '__monolithic_build__']
__all__ = [_s for _s in dir() if not _s.startswith('_')]
__all__.extend([_s for _s in _names_with_underscore])
__all__.remove('print_function')


from tensorflow.python.util.lazy_loader import LazyLoader  # pylint: disable=g-import-not-at-top
contrib = LazyLoader('contrib', globals(), 'tensorflow.contrib')
del LazyLoader

from tensorflow.python.platform import flags  # pylint: disable=g-import-not-at-top
app.flags = flags  # pylint: disable=undefined-variable

del absolute_import
del division
del print_function

# These symbols appear because we import the python package which
# in turn imports from tensorflow.core and tensorflow.python. They
# must come from this module. So python adds these symbols for the
# resolution to succeed.
# pylint: disable=undefined-variable
del python
del core
# pylint: enable=undefined-variable
```
After installation of tf-serving-api:
```
[root@eac5952e1443 /]# cat /usr/local/anaconda3/lib/python3.6/site-packages/tensorflow/__init__.py 
[root@eac5952e1443 /]# 
```

**Describe the expected behavior**
```
>>> print(tf.__version__)
1.10.0
```
**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.
```
pip install tensorflow==1.10
pip install tensorflow-serving-api==1.10
python
>> import tensorflow as tf
>> print(tf.__version__)
```

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

Of course I know that it would work fine if I reinstall tensorflow after tf-serving-api installation. But I need to install all of these into a docker image from Dockerfile. Reinstall seems not to be a good choice. Anyone has any suggestions? Thanks.

**Comment by mihaimaruseac** [comment]
TensorFlow 1.10 is long outside of the support window as it has been released more than a year ago.

Please try using a newer version and let us know if issue persists.

**Comment by gowthamkpr** [comment]
@mihaimaruseac Using TF 1.14 with TF Serving 1.14, doesn't result in any error. Please find the colab notebook [here](https://colab.sandbox.google.com/gist/gowthamkpr/b915a82d399c2e1a91f035574e87c93c/untitled91.ipynb)

**Comment by gowthamkpr** [comment]
@elvys-zhang Can I close the issue as it has been resolved. Thanks!

**Comment by elvys-zhang** [comment]
@gowthamkpr Sure. I found that this issue has been fixed in tensorflow serving 1.10.1. Thanks
